src/Corrade/Utility/Directory.cpp

```diff
@@ -202,23 +202,39 @@ bool mkpath(const std::string& path) {
     if(path.back() == '/')
         return mkpath(path.substr(0, path.size()-1));
 
-    /* If the directory exists, done */
-    if(exists(path)) return true;
-
     /* If parent directory doesn't exist, create it */
     const std::string parentPath = Directory::path(path);
-    if(!parentPath.empty() && !exists(parentPath) && !mkpath(parentPath)) return false;
+    if(!parentPath.empty() && !exists(parentPath) && !mkpath(parentPath)) {
+        Error{} << "Utility::Directory::mkpath(): error: could not create parent path";
+        return false;
+    }
```

Comment:
Assuming `mkpath(parentPath)` already prints an error message, I think this message is superfluous :)

Reply:
✅

Reply:
Yeah so this change could be completely reverted ;)